CharlieBackend.Business/Services/Interfaces/IHomeworkStudentService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using CharlieBackend.Core.DTO.HomeworkStudent;
+using CharlieBackend.Core.Models.ResultModel;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Business.Services.Interfaces
+{
+    public interface IHomeworkStudentService
+    {
+        Task<Result<HomeworkStudentDto>> CreateHomeworkFromStudentAsync(HomeworkStudentRequestDto homeworkStudent);
+
+        Task<IList<HomeworkStudentDto>> GetHomeworkStudentForMentorByHomeworkId(long homeworkId);
+
+        Task<IList<HomeworkStudentDto>> GetHomeworkStudentForStudent();
```

Comment:
```suggestion
        Task<IList<HomeworkStudentDto>> GetHomeworkForMentor(long homeworkId);

        Task<IList<HomeworkStudentDto>> GetHomeworkForStudent();
```
Inconsistency: in one case you're adding suffix `ByHomeworkId`, in other case you don't. I think it is redundant.